lua/null-ls/health.lua

```diff
@@ -0,0 +1,43 @@
+local health = require("health")
+local sources = require("null-ls.sources")
+local utils = require("null-ls.utils")
+
+local M = {}
+
+local messages = {
+    ["executable"] = [[%s: the command "%s" is executable.]],
+    ["not-executable"] = [[%s: the command "%s" is not executable.]],
+    ["unable"] = [[%s: cannot verify if the command is an executable.]],
+    ["executable-local"] = [[%s: the command "%s" is not globally executable, but it may be available locally.]],
+}
+
+local function report(source)
+    local name = source.name
+    local only_local = source.generator.opts.only_local
+    local command = source.generator.opts.command
```

Comment:
```suggestion
    local only_local = source.generator.opts and source.generator.opts.only_local
    local command = source.generator.opts and source.generator.opts.command
```

Since `opts` is just a convention used by built-ins, we need to guard against the possibility that `opts` is nil here.